src/components/general-components/MapCard.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Card } from 'antd';
+import './general-component-style/map-card.less';
+
+type MapCardProps = {
+  cardHeader: string;
+  cardBody: string;
+};
+
+const MapCard: React.FC<MapCardProps> = (props) => {
```

Comment:
```suggestion
const MapCard: React.FC<MapCardProps> = ({
  cardHeader,
  cardBody,
}) => {
```
Preference - destructure the props inline